packages/block-library/src/query-total/block.json

```
@@ -25,7 +25,8 @@
 			"gradients": true,
 			"text": true,
 			"__experimentalDefaultControls": {
```

```suggestion
			"__experimentalDefaultControls": {
```

Nit: I think we can remove this duplicate "text" field.

---

@HILAYTRIVEDI I would like to commit this suggestion so that this PR is shipped to Beta2.